docs/release-notes/index.rst

```diff
@@ -6,6 +6,8 @@ Release notes
 .. toctree::
     :titlesonly:
 
+    whats-new-3
     whats-new-2
+    3.x Changelog <https://litestar.dev/3-dev/release-notes/changelog.html>
```

Comment:
Will we be starting a new changelog document for the 3.x series, or just continue on with the existing changelog and add the 3.x stuff at the top? At the moment, this points to the 2.x changelog document?

Reply:
New one for 3.0 preferably

Reply:
And does the 2.x changelog still exist in the 3.0 branch, or do I just delete everything from the existing one and change the document title?